src/app/modules/register-login/components/register/register.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { User } from 'src/app/models/user';
+import { AuthService } from 'src/app/services/auth.service';
+import { Router } from '@angular/router';
+import { SnacbarAlertService } from 'src/app/services/snacbar-alert.service';
+import { NgxSpinnerService } from 'ngx-spinner';
+
+const REQUIRES_MESSAGE = 'You must enter a value';
+const NOTVALID_EMAIL_MESSAGE = 'Not a valid email';
+const BLANK_MESSAGE = '';
+const NOTVALID_PASSWORDMATCHES_MESSAGE = 'Password dont matches';
```

Comment:
Typo: lack of apostrophe in word don't

Reply:
fixed by:
https://github.com/anowakowski/PictureApp_SPA/pull/25/commits/c756011b6cfe7fe12c9a6b15ddaa543cb1ec0117